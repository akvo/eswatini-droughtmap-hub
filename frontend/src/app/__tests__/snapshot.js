import { render, waitFor } from "@testing-library/react";
import Home from "../page";

it("renders homepage unchanged", async () => {
  await waitFor(() => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
