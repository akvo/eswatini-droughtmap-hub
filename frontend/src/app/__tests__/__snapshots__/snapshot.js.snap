// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="w-full h-screen flex items-start justify-between"
  >
    <div
      class="w-full h-full lg:w-1/2"
    >
      <div
        role="figure"
      />
    </div>
    <div
      class="w-full block lg:w-1/2 p-2 space-y-6 p-2"
    >
      <textarea
        id="tiny-react_58606498511736232380034"
        style="visibility: hidden;"
      />
      <hr />
      <div />
    </div>
  </div>
</div>
`;
